service_container/service_subscribers_locators.rst

```diff
@@ -346,22 +352,23 @@ services::
     use App\CommandHandler\FooHandler;
     use Psr\Container\ContainerInterface;
     use Symfony\Component\DependencyInjection\Attribute\AutowireLocator;
+    use Symfony\Component\DependencyInjection\Attribute\SubscribedService;
 
     class CommandBus
     {
         public function __construct(
-            #[AutowireLocator(
-                fooHandlerAlias: FooHandler::class,
-                barHandlerAlias: BarHandler::class,
-                optionalBazHandlerAlias: '?'.BazHandler::class
+            #[AutowireLocator([
+                'fooHandlerAlias' => FooHandler::class,
+                'barHandlerAlias' => new SubscribedService(type: BarHandler::class, nullable: true),
```

Comment:
@kbond you mean like this?

Reply:
This example is valid but doesn't show the value of using `SubscribedService`. It can be used to add additional DI attributes (in the example linked above, it uses `Autowire`).